[2023-11-06T12:06:06.528+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-11-01T00:00:00+00:00 [queued]>
[2023-11-06T12:06:06.545+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-11-01T00:00:00+00:00 [queued]>
[2023-11-06T12:06:06.546+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:06:06.568+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-11-01 00:00:00+00:00
[2023-11-06T12:06:06.574+0000] {standard_task_runner.py:57} INFO - Started process 889 to run task
[2023-11-06T12:06:06.579+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-11-01T00:00:00+00:00', '--job-id', '106', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmpqf69ebr_']
[2023-11-06T12:06:06.583+0000] {standard_task_runner.py:85} INFO - Job 106: Subtask task_a
[2023-11-06T12:06:06.655+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-11-01T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:06:06.763+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-11-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-01T00:00:00+00:00'
[2023-11-06T12:06:06.765+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:06:06.766+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World :)']
[2023-11-06T12:06:06.776+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:06:06.779+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: syntax error near unexpected token `)'
[2023-11-06T12:06:06.780+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: `echo Hello World :)'
[2023-11-06T12:06:06.780+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-11-06T12:06:06.795+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-11-06T12:06:06.800+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231101T000000, start_date=20231106T120606, end_date=20231106T120606
[2023-11-06T12:06:06.817+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 106 for task task_a (Bash command failed. The command returned a non-zero exit code 2.; 889)
[2023-11-06T12:06:06.833+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-06T12:06:06.857+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-06T12:19:12.793+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-11-01T00:00:00+00:00 [queued]>
[2023-11-06T12:19:12.814+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-11-01T00:00:00+00:00 [queued]>
[2023-11-06T12:19:12.816+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:19:12.852+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-11-01 00:00:00+00:00
[2023-11-06T12:19:12.860+0000] {standard_task_runner.py:57} INFO - Started process 2102 to run task
[2023-11-06T12:19:12.867+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-11-01T00:00:00+00:00', '--job-id', '178', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmpto5myzuu']
[2023-11-06T12:19:12.874+0000] {standard_task_runner.py:85} INFO - Job 178: Subtask task_a
[2023-11-06T12:19:12.981+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-11-01T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:19:13.224+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-11-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-01T00:00:00+00:00'
[2023-11-06T12:19:13.227+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:19:13.228+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World.']
[2023-11-06T12:19:13.248+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:19:13.249+0000] {subprocess.py:93} INFO - Hello World.
[2023-11-06T12:19:13.250+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-11-06T12:19:13.302+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231101T000000, start_date=20231106T121912, end_date=20231106T121913
[2023-11-06T12:19:13.347+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-06T12:19:13.381+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
